listenbrainz/webserver/__init__.py

```diff
@@ -82,11 +83,7 @@ def create_app(debug=None):
     if app.debug and app.config['SECRET_KEY']:
         app.init_debug_toolbar()
 
-    # Logging
-    app.init_loggers(
-        file_config=app.config.get('LOG_FILE'),
-        sentry_config=app.config.get('LOG_SENTRY')
-    )
+    init_sentry(**app.config.get('LOG_SENTRY'))
```

Comment:
```suggestion
    sentry.init_sentry(**app.config.get('LOG_SENTRY'))
```
with the associated change to import `sentry` as a module instead of the plain method (alongside cache/metrics)

Reply:
Fixed alongside checking if config is not None.